feature_engine/encoding/base_encoder.py

```diff
@@ -79,10 +79,14 @@ def __init__(
         variables: Union[None, int, str, List[Union[str, int]]] = None,
         ignore_format: bool = False,
         errors: str = "ignore",
+        supports_errors_encode: bool = False
```

Comment:
hi @david-cortes 

We can't really add parameters unless they are strictly necessary, because that turns classes less user friendly.

I am trying to think what would be the best way forward, given that at the moment, all classes take "raise" and "ignore" and we want the Count frequency to take the extra parameter.

I think, probably the best way is to take this functionality out of the base class and into a function, that takes the allowed strings as inputs.

Unless you have a different idea?

Reply:
Yes, that also would do.